css/css-ui/appearance-auto-ref.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Reference: appearance: auto</title>
+<style>
```

Comment:
That reference does not contain `appearance: auto`, and instead relies on the UA stylesheet having that value (or some other value with the same effect). Which in practice will be the case, but isn't yet (as far as I know) in any spec, and is a separate thing to test anyway. Please explicitly add `appearance: auto` to this ref file, since it isn't the initial value of the property. Otherwise, we're testing the content of the UA stylesheet, which is a valid, but separate, thing to do.